content-review/impl/turnitin-oc/src/main/resources/turnitin_es.properties

```diff
@@ -0,0 +1,18 @@
+file.type.doc=Word 97-2003
```

Comment:
Are these supposed to be translated or are they supposed to be static? Are these called the same thing in Spanish?

Reply:
In Spanish is the same thing.